### PR TITLE
Add information about gas increase for FIP-0045

### PR DESCRIPTION
**Pull request by Kubuxu**

### CODE REVIEW

File: FIPS/fip-0045.md [hunk 1]
```diff
@@ -779,7 +779,9 @@ the final decoupling from the miner actor.
   including those far more efficient than the built-in one.
 
 By opening the field of competition for market-like actors, we can expect deal costs to decrease over time,
-while the network enforces just enough state to administer the quality-adjusted power of verified data. 
+while the network enforces just enough state to administer the quality-adjusted power of verified data.'
+
+Initial evaluations of changes introduced suggest gas usage increase of 175%.
```
Comment (Stebalien) [inline FIPS/fip-0045.md:784]: Can we specify "for what"? I assume this isn't 175% across the board.

Comment (jennijuju) [inline FIPS/fip-0045.md:784]:
```suggestion
Initial evaluations of changes introduced suggest gas usage increase of 175% for a `PublishStorageDeals` message that has 8 verified deals.
```

Comment (Kubuxu) [reply FIPS/fip-0045.md:784]: Thanks, I was so deep into testing the PSD that I forgot that there are other things :P

**Comment by jennijuju** [comment]
@Kubuxu shall we close this in favor of #523 ?

**Comment by steven004** [comment]
You may want to update the `Implementation` section, which is out of date since the branch is deleted after merge.

**Comment by kaitlin-beegle** [comment]
Closing PR as we await new, updated gas estimates.

**Pull request closed without merging**